tool/teleport/common/teleport.go

```diff
@@ -96,7 +96,7 @@ func Run(options Options) (app *kingpin.Application, executedCommand string, con
 	join := app.Command("join", "Join a Teleport cluster without running the Teleport daemon.")
 	joinOpenSSH := join.Command("openssh", "Join an SSH server to a Teleport cluster.")
 	scpc := app.Command("scp", "Server-side implementation of SCP.").Hidden()
-	sftp := app.Command("sftp", "Server-side implementation of SFTP.").Hidden()
+	sftp := app.Command(teleport.SFTPSubCommand, "Server-side implementation of SFTP.").Hidden()
```

Comment:
Personally, I'd leave this the way it was.

I often try to find the entry point for a particular subcommand by searching for `.Command("foo`, which no longer works if the subcommand is replaced with an unknown variable name.

Reply:
I went at this from the exact opposite angle: I couldn't find the `app.Command` entry by searching references to `teleport.SFTPSubCommand` and I had to go fishing for the string literal again; I'd rather leave it like this, since the other reexecs (other than scp, for whatever reason?) are also defined through their `FooSubCommand` const, and `"sftp"` could refer to both the subcommand name and the subsystem argument.